guide/principles/principles.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: guide
+title: Bitcoin design principles
+description: The key principles to follow when designing Bitcoin products.
+nav_order: 2
+has_children: false
+permalink: /guide/principles/principles/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+The key principles of designing for Bitcoin
+
+-->
+
+# Bitcoin design principles
+
+As a new technology, Bitcoin offers the opportunity of a decentralized open financial system without intermediaries. To make this a reality, we encourage everyone that is working on products to deliberately support the core principles of designing for Bitcoin. Some of these come from the technology itself, and others from the behaviour and ethos of the community. Although every use case and product is different, all applications should strive to follow these principles. Diverging from them should only be done with very good reason.
+
+
+- [Decentralization](#decentralization)
+- [Inclusion](#inclusion)
+- [Transparency](#transparency)
+- [Self-custody](#self-custody)
+- [Security](#security)
+- [Privacy](#privacy)
+- [Interoperability](#interoperability)
+
+---
+
+## Decentralization
+
+Unlike traditional banking systems, the Bitcoin economy does not require new users to seek permission from anyone. Bitcoin has no central point of control. No one person or entity is in charge. Connecting to any node on the network gives you the same rights and responsibilities, which also ensures that there is no single point of failure.
+
+*Build products that are permissive in terms of what users can do with Bitcoin.* <br>
+*Have fall-back options for connections to the network.*
+
+**Do**
+- Manage your own private keys
+- Run a full Bitcoin node when possible
+- Alternatively, use a light client with the [p2p network]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki'}}) using [compact block filters]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki'}}) 
+- Offer the user choice of what node to connect to
```

Comment:
Wonder if this is a larger point than just nodes? Should I be able to choose an explorer, or where to get currency conversion data from? All external connections can leak meta data.

Reply:
Added 'and other external services'. 7a7d5c2